spec/domain/time_entry/description_rules_spec.rb

```diff
@@ -45,30 +45,12 @@
         expect(validator.valid?).to eql(true)
       end
 
-      it "will pass if it has a keyword" do
-        entry = Entry.new(description: "This is an English class entry")
-        validator = TimeEntry::DescriptionRules.new(entry)
-        expect(validator.valid?).to eql(true)
-
```

Comment:
Why we are removing these tests?

Reply:
Thanks for catching this bc the intention was not to remove these tests so I will have to double check!